internal/client/blueprints_test.go

```diff
@@ -1006,6 +1006,35 @@ func TestBlueprintDepsolveGlobsV0(t *testing.T) {
 	assert.True(t, common.IsStringInSortedSlice(names, "tmux"))
 }
 
+// depsolve a blueprint with package from 3rd party with module_hotfixes
+func TestBlueprintDepsolveModuleHotfixesV0(t *testing.T) {
+	// Depends on real packages, only run as an integration test
+	if testState.unitTest {
+		t.Skip()
+	}
+	bp := `{
+		"name": "test-deps-blueprint-module-hotfixes-v0",
+		"description": "CheckBlueprintDepsolveModuleHotfixesV0",
+		"version": "0.0.1",
+		"packages": [{"name": "nginx", "version": "*"},
+		{"name": "nginx-module-njs", "version": "*"}]
+		"customizations": {"repositories": [{
+			"id": "nginx", "name": "nginx", "baseurls": ["https://rpmrepo.osbuild.org/v2/mirror/public/el8/el8-x86_64-nginx-20231207"], "gpgcheck"": false}
```

Comment:
You should set module_hotfixes to false to make it clear what's happening here.
Also, shouldn't you also test it with it set to true?

Reply:
oh, sorry it was not really ready, I supposed to have drafted it :)
Now it's what I believe we should add, but it still doesn't work locally, I think it uses a my local dnfjson maybe? :)

Reply:
Well, I've pushed next rebase, let see what happens :shrug:
If it stays red, I'm out of ideas :thinking: